packages/react-celo/src/connectors/injected.ts

```diff
@@ -32,23 +32,29 @@ export default class InjectedConnector
     this.network = network;
   }
 
-  async initialise(): Promise<this> {
+  async initialise(lastUsedAddress?: string): Promise<this> {
     if (this.initialised) {
       return this;
     }
 
+    // is this a bad idea?
+    let defaultAccount = lastUsedAddress;
```

Comment:
Idk, it feels like a bad idea but it probably isn't

Reply:
yeah